host-controller/src/main/java/org/jboss/as/host/controller/DomainModelControllerService.java

```diff
@@ -888,6 +893,7 @@ public void stop(final StopContext context) {
             }
         }
         extensionRegistry.clear();
+        capabilityRegistry.clear();
```

Comment:
Can this move to AbstractControllerService? It would be nice if the clear() method wasn't public.